website/docs/core-concepts/connecting-to-data-sources/README.md

```diff
@@ -1,5 +1,41 @@
 # Connecting to Datasource
 
+## **Outline**
+
+**Introduction** 
+
+**Prerequisites​**
+
+Before you begin, you'll need to create an account on Appsmith Cloud, or you can self-host Appsmith locally or on a hosting platform of your choice.
+
+**How to connect to a datasource**
+
+Explain how to connect to a datasource
+
+**APIs**
+* Information about APIs
+* List of supported APIs. 
+* Sample API​ (Explain how to use sample API)
+
+**Databases**
+* Information about databases
+* List of supported Database.
+* Sample DB​ (Explain how to use sample DB, both movies and users)
+
+**Security**
```

Comment:
This section looks a bit offbeat to me. Because we are not talking about the datasource security setups here. It needn't be a section - just a callout or highlight to read more on security.

Reply:
I agree. we dont need to mention it at all. We have a Security page for this, if users want to check it out